src/components/structures/LegacyCallEventGrouper.ts

```diff
@@ -37,7 +37,7 @@ const CONNECTING_STATES = [
 const SUPPORTED_STATES = [CallState.Connected, CallState.Ringing, CallState.Ended];
 
 const isCallEventType = (eventType: string): boolean =>
-    eventType.startsWith("m.call.") || eventType.startsWith("org.matrix.call.");
+    eventType?.startsWith("m.call.") || eventType?.startsWith("org.matrix.call.");
```

Comment:
This type of change is suspicious to me - we should be able to have full trust in our types when we say that `eventType` has type `string`